test/parallel/test-tls-timeout-server-2.js

```diff
@@ -1,32 +1,32 @@
 'use strict';
-var common = require('../common');
-var assert = require('assert');
+const common = require('../common');
+const assert = require('assert');
 
 if (!common.hasCrypto) {
   common.skip('missing crypto');
   return;
 }
-var tls = require('tls');
+const tls = require('tls');
 
-var fs = require('fs');
+const fs = require('fs');
 
-var options = {
+const options = {
   key: fs.readFileSync(common.fixturesDir + '/keys/agent1-key.pem'),
   cert: fs.readFileSync(common.fixturesDir + '/keys/agent1-cert.pem')
 };
 
-var server = tls.createServer(options, function(cleartext) {
-  var s = cleartext.setTimeout(50, function() {
+const server = tls.createServer(options, common.mustCall(function(cleartext) {
+  const s = cleartext.setTimeout(50, common.mustCall(function() {
```

Comment:
No worries, I'm sure there's been a lot of email with all of the Code & Learn commits going around.

I just pushed a commit that removes this inner `common.mustCall`. For consistency, would you like me to submit a separate PR that also removes something very similar from [`test-tls-timeout-server`](https://github.com/nodejs/node/blob/8264a2225628cbc32116f83a079f6024786988ab/test/parallel/test-tls-timeout-server.js#L21)?